service/src/main/resources/supported-content-types.txt

```diff
@@ -0,0 +1,33 @@
+application/msword
+application/pdf
+application/xml
+application/vnd.ms-excel.addin.macroEnabled.12
```

Comment:
Are we able to produce correct file extensions for each of these types?

Reply:
yes, the xml file with mappings contains all of them